docker/ssh_config

```diff
@@ -2,3 +2,5 @@ Host *
     User aeternity
     IdentityFile ~/.ssh/id_ae_infra_ed25519
     CertificateFile ~/.ssh/id_ae_infra_ed25519-%r-cert.pub
+    ServerAliveInterval 120
```

Comment:
I think this is an option for the sshd_config (ssh server) while for the ssh_config (ssh client) it should be `ClientAliveInterval`